Sources/Overlays/_Testing_CoreGraphics/Attachments/_AttachableImageWrapper.swift

```diff
@@ -61,7 +62,7 @@ public struct _AttachableImageWrapper<Image>: Sendable where Image: AttachableAs
   var encodingQuality: Float
 
   /// Storage for ``contentType``.
-  private var _contentType: (any Sendable)?
+  private var _contentType: UTType?
```

Comment:
Could we consolidate this property with `contentType` now?

Reply:
No, see the additional logic in `_contentType`.